<!-- DESCRITPION -->

<launch>
	<node pkg="ira_laser_tools" name="laserscan_multi_merger" exec="laserscan_multi_merger" output="screen">
		<param name="destination_frame" value="base_link"/>
		<param name="cloud_destination_topic" value="/merged_cloud"/>
		<param name="scan_destination_topic" value="/scan_multi"/>
		<param name="laserscan_topics" value ="/scansx /scandx" />
		<!-- LIST OF THE LASER SCAN TOPICS TO SUBSCRIBE -->
		<param name="angle_min" value="-3.14"/>
		<param name="angle_max" value="3.14"/>
		<param name="angle_increment" value="0.00437"/>
		<param name="scan_time" value="0.0"/>
		<param name="range_min" value="0.1"/>
		<param name="range_max" value="2.0"/>
	</node>
</launch>
